xml/art_sles_rpi_quick.xml

```diff
@@ -1096,8 +939,213 @@ Device setup complete</screen>
+    </para>
+   </note>
+  </sect2>
+
+  <sect2 xml:id="sec.rpi.hostname">
+   <title>Changing the Host Name</title>
+   <para>
+    Follow these steps to change the default host name:
+   </para>
+   <procedure xml:id="pro.rpi.hostname.yast">
+    <step>
+     <para>
+      Open the &yast; network module either by running <command>yast2
+      lan</command> or clicking the network icon in the &yast; control center.
+     </para>
+    </step>
+    <step>
+     <para>
+      Select <guimenu>Hostname/DNS</guimenu> in the top tab bar.
+     </para>
+    </step>
+    <step>
+     <para>
+      Type the new host name into the <guimenu>Hostname</guimenu> field.
+     </para>
+    </step>
+    <step>
+     <para>
+      Click <guimenu>OK</guimenu> to save the change. After &yast; exited, you
+      need to log out and in again.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.rpi.network">
+   <title>Setting up Networking</title>
+   <para>
+    The default configuration has DHCP enabled on the Ethernet port. If that
+    suits your network environment, you can skip this section. If you require
+    the use of a static IP address, use &yast;:
+   </para>
+   <procedure xml:id="pro.rpi.network.yast">
+    <step>
+     <para>
+      Open the &yast; network module either by running <command>yast2
+      lan</command> or clicking the network icon in the &yast; control center.
+     </para>
+    </step>
+    <step>
+     <para>
+      In &yast;, you will see the network interface being selected. Select the
+      built-in Ethernet and choose <guimenu>Edit</guimenu> to open the address
+      configuration.
+     </para>
+    </step>
+    <step>
+     <para>
+      Select <guimenu>Statically Assigned IP Address</guimenu> and type in the
+      desired values for <guimenu>IP Address</guimenu> and <guimenu>Subnet
+      Mask</guimenu>. Click <guimenu>Next</guimenu>.
+     </para>
+    </step>
+    <step>
+     <para>
+      With a static network configuration, you will also need to specify a DNS
+      server (if applicable) and a gateway. For the gateway, select
+      <guimenu>Routing</guimenu> in the tab bar and enter the IPs of the
+      gateways into the specific fields.
+     </para>
+    </step>
+    <step>
+     <para>
+      The DNS server is set in the <guimenu>Hostname/DNS</guimenu> tab. After
+      choosing that tab, enter the IPs of the name servers into the respective
+      <guimenu>Name Server</guimenu> fields.
+     </para>
+    </step>
+   </procedure>
+   <para>
+    Similarly to the procedure described above, &yast; also lets you configure
+    the built-in Wi-Fi network adapter.
+   </para>
+   <para>
+    For detailed information about the network configuration in &productnamearch;,
+    consult the respective sections of the
+    <link
+          xlink:href="https://www.suse.com/documentation/sles-12/book_sle_deployment/data/book_sle_deployment.html"
```

Comment:
URL labels again...I know, you see these all over. We're working on correcting all of them.